Framework/Kernel/src/FileValidator.cpp

```diff
@@ -6,9 +6,9 @@
 // SPDX - License - Identifier: GPL - 3.0 +
 #include "MantidKernel/FileValidator.h"
 #include "MantidKernel/Logger.h"
-#include <Poco/File.h>
 #include <Poco/Path.h>
```

Comment:
Header not required

Reply:
I was unable to find a replacement for `Poco::Path().tryParse(value)`